journal-replace/modernized/29-section-28.adoc

```diff
@@ -1,7 +1,7 @@
 == Section XXVIII.
 
-In several meetings, both National and Provincial, he loas earnest to have marriage,
```

Comment:
I'm not sure if "loas" is right here? I thought it might be "was" but didn't want to change it without the PDF to look it up.